packages/apps/shopify-app-remix/docs/staticPages/webhooks.doc.ts

```diff
@@ -9,15 +9,37 @@ const data: LandingTemplateSchema = {
     {
       type: 'Generic',
       anchorLink: 'config',
-      title: 'Configuring webhooks subscriptions',
+      title: 'Subscribe using the app configuration file',
+      sectionContent:
+        'The easiest and recommended way to configure your webhooks is to edit your app configuration file. ' +
+        'You can find more info in the [webhooks documentation](/docs/apps/webhooks/getting-started-declarative).' +
+        '\n\nTo set up a simple HTTPS webhook subscription, you can follow these steps:' +
+        '\n1. Add in `shopify.app.toml` the topic you want to subscribe to. In this example we subscribe to the `APP_UNINSTALLED` topic.' +
+        '\n1. Review the required scopes for the webhook topics, and update your [app scopes](/docs/apps/tools/cli/configuration#access_scopes) as necessary.' +
```

Comment:
Do we cover PubSub / EventBridge events in this page? If not, do we want to mention it somewhere in these docs?